spec/src/main/asciidoc/metrics.asciidoc

```diff
@@ -212,43 +236,57 @@ public class MyClass {
 
 This class would result in the following metrics being added.
 
-* `ft.com.example.MyClass.doWork.invocations.total`
-* `ft.com.example.MyClass.doWork.invocations.failed`
-* `ft.com.example.MyClass.doWork.retry.callsSucceededNotRetried.total`
-* `ft.com.example.MyClass.doWork.retry.callsSucceededRetried.total`
-* `ft.com.example.MyClass.doWork.retry.callsFailed.total`
-* `ft.com.example.MyClass.doWork.retry.retries.total`
-* `ft.com.example.MyClass.doWork.timeout.executionDuration`
-* `ft.com.example.MyClass.doWork.timeout.callsTimedOut.total`
-* `ft.com.example.MyClass.doWork.timeout.callsNotTimedOut.total`
+```
+ft.invocations.total{method="com.example.MyClass.doWork", result="valueReturned", fallbackUsed="none"}
+ft.invocations.total{method="com.example.MyClass.doWork", result="exceptionThrown", fallbackUsed="none"}
```

Comment:
Old name `fallbackUsed`.

Reply:
ah rats, thank you for spotting this, I completely forgot to update the example.